nifi-registry/pom.xml

```diff
@@ -356,6 +356,11 @@
             <artifactId>slf4j-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+            <version>1.5.16</version>
+        </dependency>
```

Comment:
Adding the swagger-annotations dependency at the parent level appears to be adding it to all modules, however, it seems like it should only be necessary in the `nifi-registry-data-model` module.  Is there a reason for including here?